clang-tools-extra/test/clang-tidy/checkers/misc/include-cleaner.cpp

```diff
@@ -15,3 +15,5 @@ std::string HelloString;
 // CHECK-MESSAGES: :[[@LINE-1]]:6: warning: no header providing "std::string" is directly included [misc-include-cleaner]
 int FooBarResult = foobar();
 // CHECK-MESSAGES: :[[@LINE-1]]:20: warning: no header providing "foobar" is directly included [misc-include-cleaner]
+
+void log2() {}
```

Comment:
Maybe write a small comment stating it shouldn't warn, or add a reference to the issue number? That would make it easier to understand why this is here